main/commons/src/main/java/org/cryptomator/common/settings/VaultSettingsJsonAdapter.java

```diff
@@ -29,7 +29,7 @@ public void write(JsonWriter out, VaultSettings value) throws IOException {
 		out.name("customMountPath").value(value.customMountPath().get());
 		out.name("usesReadOnlyMode").value(value.usesReadOnlyMode().get());
 		out.name("mountFlags").value(value.mountFlags().get());
-		out.name("filenameLengthLimit").value(value.filenameLengthLimit().get());
+		out.name("maxCleartextFilenameLength").value(value.maxCleartextFilenameLength().get());
```

Comment:
Do we need some migration from "old" settings? Or will this new setting automatically be set to `-1` and then the capability check will be repeated during the first unlock?

Reply:
No, we **do not** want a migration. Since the old value has a different meaning (i.e. max ciphertext length), we must make sure **not** to use it for the new (cleartext) limit. Therefore we just ditched the old json field (will be ignored upon reading the settings file) and introduced a new one.

Reply:
Ah, gotcha. 👍